src/features/ui/MoreNFTOptions/MoreNFTOptions.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState } from 'react';
+
+import { Option } from './MoreNFTOptions.types';
+import { OptionType } from './MoreNFTOptions.constants';
+
+import { TransferOwnershipModal } from '../../transfer-ownership';
+import { DropdownMenu, DropdownMenuProps } from '@zero-tech/zui/components';
+import { useMoreOptions } from './hooks/useMoreOptions';
+
+type MoreNFTOptionsProps = {
+	domainId: string;
+	domainName: string;
+	domainTitle: string;
+	domainOwner: string;
+	domainCreator: string;
+	trigger: DropdownMenuProps['trigger'];
```

Comment:
nit: Would be good to use `useDomainData` hook rather than passing data in, and only having a `zna` or `domainId` as props.

Reply:
Done!